src/scripts/controllers/container.controller.js

```diff
@@ -56,6 +56,9 @@
             this.ngModel = {};
           } else if (this.ngModel instanceof Date) {
             this.ngModel.setHours(0, 0, 0, 0);
+          } else if (typeof(this.ngModel) === 'string' || this.ngModel instanceof String) {
```

Comment:
Not exactly, there are specific cases that would require these two comparisons. See the example:
![image](https://cloud.githubusercontent.com/assets/228961/6728512/0953edf2-ce09-11e4-82ef-54a52f06c144.png)